draft-ietf-ppm-dap.md

```diff
@@ -624,30 +624,18 @@ This message is called the client's *report*. It contains the following fields:
 
 To generate the report, the client begins by sharding its measurement into a
 sequence of input shares as specified by the VDAF in use. To encrypt an input
-share, the client first generates an HPKE {{!RFC9180}} context for
-the aggregator by running
+share, the client generates an HPKE {{!RFC9180}} ciphertext and encapsulated
+key for the aggregator by running
 
 ~~~
-enc, context = SetupBaseS(pk, Report.task_id || "dap-01 input share" ||
-                              0x01 || server_role)
+enc, payload = SealBase(pk, "dap-01 input share" || 0x01 || server_role,
```

Comment:
Should we take the opportunity to bump this to `dap-02 input share` since we plan to submit `draft-ietf-ppm-dap-02` soon? Same goes for `dap-02 aggregate share` elsewhere in the doc.

Reply:
We're at -00 right now: https://datatracker.ietf.org/doc/draft-ietf-ppm-dap/

Reply:
Leaving as `dap-01` for now since we are currently at `dap-00` and per Tim's comment we will soon be bumping versions.